lib/inspec/resources/bash.rb

```diff
@@ -3,7 +3,7 @@
 require 'inspec/resources/command'
 
 module Inspec::Resources
-  class Bash < Cmd
+  class Bash < Command
```

Comment:
I did this one because I was tired and getting confused by cmd the windows shell vs Command, esp once it started being subclassed by wholly different shells.